src/shrdlu/blockl.6

```diff
@@ -250,13 +250,15 @@
 (DEFUN PACKO
        (OBJ TYPE)
        (PROG (XX)
+	     (setq xx nil)
```

Comment:
PROG should bind XX to NIL already (per the Pitmanual).  Is this necessary?

Reply:
I was getting an XX unbound variable here and the change appeared to fix it, but I was stumped because both the circa 197x LISP and the current LISP do bind prog variables to nil. I can try to make it through the script without that change and we can see if it made any difference.